index.html

```diff
@@ -161,6 +172,40 @@ <h3>Skills</h3>
                 </ul>
             </div>
         </section>
+        <section class="contact-form">
+            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            <form action="https://formspree.io/f/xayvgeok" method="post">
+                <ul>
+                    <li>
+                        <label for="name"></label>
+                        <input type="text" name="full-name" id="name" placeholder="Full name" required>
+                    </li>
+                    <li>
+                        <label for="mail"></label>
+                        <input type="email" name="email" id="mail" placeholder="Email address" required>
+                    </li>
+                    <li>
+                        <textarea name="message" id="message" cols="30" rows="10" placeholder="Enter text here"
+                            required></textarea>
```

Comment:
- Please, kindly add the maxlength attribute to the message input to meet the project requirements.

Reply:
@zieeco i already modified it thanks alot